### ISSUE TITLE
fib test failed due to fib test failed due to "AttributeError: TestHash instance has no attribute 'thresholds'"

### ISSUE THREAD

**Issue by Junchao-Mellanox** [Azure/sonic-mgmt#1679]
fib test failed due to "AttributeError: TestHash instance has no attribute 'thresholds'". Error information:
```
============================================================ ERRORS =============================================================
___________________________________________ ERROR at setup of TestHash.test_hash_ipv4 ___________________________________________

self = <test_fib.TestHash instance at 0x7f150403ed88>
dut_ssh = <common.plugins.dut_monitor.pytest_dut_monitor.DUTMonitorClient object at 0x7f15044afc90>
localhost = <common.devices.Localhost object at 0x7f1504a0ad10>, duthost = <common.devices.SonicHost object at 0x7f1505cea250>

    @pytest.fixture(autouse=True, scope="function")
    def dut_monitor(self, dut_ssh, localhost, duthost):
        """
        For each test item starts monitoring of hardware resources consumption on the DUT
        """
        dut_thresholds = {}
        monitor_exceptions = []
        # Start monitoring on DUT
        dut_ssh.start()
        logger.info("print self for debug: {}".format(dir(self)))
        # Read file with defined thresholds
>       with open(self.thresholds) as stream:
E       AttributeError: TestHash instance has no attribute 'thresholds'
```

**Steps to reproduce the issue:**

Run fib test.

**Describe the results you received:**

```
============================================================ ERRORS =============================================================
___________________________________________ ERROR at setup of TestHash.test_hash_ipv4 ___________________________________________

self = <test_fib.TestHash instance at 0x7f150403ed88>
dut_ssh = <common.plugins.dut_monitor.pytest_dut_monitor.DUTMonitorClient object at 0x7f15044afc90>
localhost = <common.devices.Localhost object at 0x7f1504a0ad10>, duthost = <common.devices.SonicHost object at 0x7f1505cea250>

    @pytest.fixture(autouse=True, scope="function")
    def dut_monitor(self, dut_ssh, localhost, duthost):
        """
        For each test item starts monitoring of hardware resources consumption on the DUT
        """
        dut_thresholds = {}
        monitor_exceptions = []
        # Start monitoring on DUT
        dut_ssh.start()
        logger.info("print self for debug: {}".format(dir(self)))
        # Read file with defined thresholds
>       with open(self.thresholds) as stream:
E       AttributeError: TestHash instance has no attribute 'thresholds'

```

**Describe the results you expected:**

No error.

**Comment by yvolynets-mlnx** [comment]
This issue will be reproduced only when will be used "--dut_monitor" pytest option.
This is a pytest issue, which was fixed in pytest version 5.1.2.
https://github.com/pytest-dev/pytest/issues/2270
https://github.com/pytest-dev/pytest/pull/5768

Currently in SONIC setups are used pytest version 4.6.5.
FYI the newest supported pytest version for Python2 is 4.6.10.

Easy solution to workaround this issue is to update test cases.

For example:
- Remove class 'TestHash' and convert its internal methods to a simple functions like it is for 'def test_fib'.
- Create 'timestamp' fixture and use it in: setup_hash, test_hash_ipv4, test_hash_ipv6
instead of 'self.t' call:
@pytest.fixture(scope="module")
def timestamp():
    yield datetime.now().strftime('%Y-%m-%d-%H:%M:%S')

- Instead of
'hash_keys = HASH_KEYS' and 'self.hash_keys' directly use 'HASH_KEYS'

This approach is tested, tests are running.

**Comment by Junchao-Mellanox** [comment]
@yvolynets-mlnx thanks.
